tests/legacy-cli/e2e/tests/mcp/registers-tools.ts

```diff
@@ -0,0 +1,41 @@
+import { chdir } from 'process';
+import { exec, ProcessOutput, silentNpm } from '../../utils/process';
+import assert from 'node:assert/strict';
+
+const MCP_INSPECTOR_PACKAGE_NAME = '@modelcontextprotocol/inspector-cli';
+const MCP_INSPECTOR_COMMAND_NAME = 'mcp-inspector-cli';
+
+async function runInspector(...args: string[]): Promise<ProcessOutput> {
+  const result = await exec(
+    MCP_INSPECTOR_COMMAND_NAME,
+    '--cli',
+    'npx',
+    '--no',
+    '@angular/cli',
+    'mcp',
+    ...args,
+  );
+
+  return result;
+}
+
+export default async function () {
+  await silentNpm('install', '--ignore-scripts', '-g', MCP_INSPECTOR_PACKAGE_NAME);
```

Comment:
Consider: Should this be more strongly versioned? What happens when `@modelcontextprotocol/inspector-cli` releases a new major version?